front-end/src/index.js

```diff
@@ -1,14 +1,14 @@
 import React from 'react';
-import { render } from 'react-dom';
-import { App } from './App.jsx';
+import ReactDOM from 'react-dom';
```

Comment:
Using ReactDOM.render instead of directly importing render, {render},  might be more helpful to someone unfamiliar with the code! I support the change

Reply:
I used ReactDOM only because line 7 used ReactDOM.render, not just render :)